src/store/reducers/walletConnectionsCache/walletConnectionsCache.ts

```diff
@@ -52,4 +57,12 @@ const getConnectedWallet = (state: RootState) =>
 const getPendingConnection = (state: RootState) =>
   state.walletConnectionsCache.pendingConnection;
 
-export { getWalletConnectionsCache, getConnectedWallet, getPendingConnection };
+const getPendingDAppMeerkat = (state: RootState) =>
```

Comment:
Also not sure here why we now have `getPendingDAppMeerkat` instead of `getPendingConnection` - can we not just change `getPendingConnection` instead of adding a new one?

Reply:
In the scan step, we only scan the MeerKat, nothing from the PeerConnection got called because we need the selected AID to start it first.

Reply:
But `getPendingConnection` is not used anymore. You should replace it with `getPendingDAppMeerkat` rather than adding a new one. So `getPendingConnection` can be removed.

In WalletConnectionStageTwo.tsx - we still have `dispatch(setPendingConnections(connection));` - should this be `setPendingDAppMeerkat`? Or if it's not needed we can remove the line completely.

Reply:
No, I use `setPendingDAppMeerkat` in stage 1, I will refactor and bring back the call of `getPendingConnection`